### DISCUSSION TITLE
Actions List

### DISCUSSION THREAD

**Discussion by cheadevit** [Laravel-Backpack/community-forum#481] [Q&A (Help)]
Hello Guys,

How to customize the button action in the list view like the picture below

<img width="507" alt="236619275-f1b8a1dd-fd05-451a-968f-669b49f69d64" src="https://user-images.githubusercontent.com/52439329/236846748-6c64a890-c6e1-4c83-9a8b-11a7191d7043.png">

Appreciate your help, thank you.

**Comment by jcastroa87** [comment]
Hello @cheadevit to create custom actions you need to follow steps on [our documentation](https://backpackforlaravel.com/docs/5.x/crud-operation-list-entries#buttons-1) you will be able to add buttons and force to show as dropdown, if you need to hide some actions, you can follow [Buttons API](https://backpackforlaravel.com/docs/5.x/crud-buttons#buttons-api-1) to make it done.

Cheers.

**Reply by cheadevit** [reply]
Thank you for you helping me.

**Comment by cheadevit** [comment]
@jorgetwgroup here is my application version

### PHP VERSION:
PHP 8.1.16 (cli) (built: Feb 14 2023 18:34:53) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.16, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.16, Copyright (c), by Zend Technologies

### LARAVEL VERSION:
v8.83.27@e1afe088b4ca613fb96dc57e6d8dbcb8cc2c6b49

### BACKPACK PACKAGE VERSIONS:
backpack/crud: 5.6.0
backpack/filemanager: 1.1.7
backpack/generators: 3.3.16
backpack/logmanager: v4.0.10
backpack/permissionmanager: 6.0.16
backpack/pro: 1.6.4
backpack/revise-operation: 1.0.12
backpack/settings: 3.0.16

**Reply by jcastroa87** [reply]
Hello @cheadevit as i send you our documentation before is cover all about Backpack v5, so let me know if you was able to customize action buttons.

Thanks.
